projects/gsc/experiments/gsc_vdrop_pruning.py

```diff
@@ -0,0 +1,513 @@
+            ),
+            dict(
+                name="max_lr",
+                type="double",
+                bounds=dict(min=0.001, max=1.5),
+                transformation="log",
+            ),
+            dict(
+                name="reg_scalar_max_value",
+                type="double",
+                bounds=dict(min=0.01, max=0.3),
+                transformation="log",
+            ),
+            dict(
+                name="reg_scalar_final_value",
+                type="double",
+                bounds=dict(min=0.0001, max=0.005),
+            ),
+            dict(
+                name="momentum_max_value", type="double", bounds=dict(min=0.6, max=0.9)
+            ),
+        ],
+        metrics=[dict(name="mean_accuracy", objective="maximize")],
+        parallel_bandwidth=1,
+        observation_budget=20,
+        project="gsc_snr_pruning",
+    ),
+    sigopt_experiment_id=374528,
+    api_key=os.environ.get("SIGOPT_KEY", None),)
+
+SIGOPT_GSC_VDROP_SNR_PRUNING_SCALING_STUDY = deepcopy(SIGOPT_GSC_VDROP_SNR_PRUNING)
+SIGOPT_GSC_VDROP_SNR_PRUNING_SCALING_STUDY.update(
+    name="SIGOPT_GSC_VDROP_SNR_PRUNING_SCALING",
+    wandb_args=dict(
+        project="gsc-snr-pruning-scaling",
+        name="scaling_study",
+    ),
+    model_class=gsc_lenet_vdrop_sparse_scaling,
+    sigopt_experiment_class=SNRPruningGSCVDropSIGOPT,
+    sigopt_config=dict(
+        name="sigopt_vdrop_gsc_snr_pruning_scaling",
+        parameters=[
+            dict(name="num_pruning_iterations", type="int", bounds=dict(min=3, max=10)),
+            dict(
+                name="epochs_per_pruning_cycle", type="int", bounds=dict(min=2, max=8)
+            ),
+            dict(
+                name="max_lr",
+                type="double",
+                bounds=dict(min=0.001, max=0.3),
+                transformation="log",
+            ),
+            dict(
+                name="reg_scalar_max_value",
+                type="double",
+                bounds=dict(min=0.01, max=1.0),
+                transformation="log",
+            ),
+            dict(
+                name="reg_scalar_final_value",
+                type="double",
+                bounds=dict(min=0.0001, max=0.005),
+                transformation="log",
+            ),
+            dict(
+                name="momentum_max_value", type="double", bounds=dict(min=0.6, max=0.9)
+            ),
+        ],
+        metrics=[dict(name="max_accuracy", objective="maximize")],
+        parallel_bandwidth=1,
+        observation_budget=120,
+        project="gsc_snr_pruning",
+    ),
+    sigopt_experiment_id=376134,
+    api_key=os.environ.get("SIGOPT_KEY", None),
+    num_samples=1,)
+
+
+class ScalingSparseBaselineExperiment(
+    mixins.LogEveryLoss,
+    mixins.LogEveryLearningRate,
+    mixins.ExtraValidationsPerEpoch,
+    mixins.ReportMaxAccuracy,
+    experiments.SupervisedExperiment,
+):
+    pass
```

Comment:
This won't train a sparse model. It needs to use `mixins.RezeroWeights` and `mixins.UpdateBoostStrength`.